sympy/sets/tests/test_setexpr.py

```diff
@@ -182,106 +182,125 @@ def test_Interval_arithmetic():
     assert n23cc / i12cc == SetExpr(Interval(-2, 3))
 
 
+def test_Interval_inf():
+    neinf = SetExpr(-oo)
+    poinf = SetExpr(oo)
+    set_ff = SetExpr(Interval(0, 5))
+    set_if = SetExpr(Interval(-oo, 5))
+    set_fi = SetExpr(Interval(0, oo))
+    set_ii = SetExpr(Interval(-oo, oo))
+
+    assert set_ff + neinf == set_if
+    assert set_ff + poinf == set_fi
+    assert set_fi + neinf == set_ii
+    assert set_if + poinf == set_ii
+
+    assert set_ff - poinf == set_if
+    assert set_ff - neinf == set_fi
+    assert set_if - neinf == set_ii
+    assert set_fi - poinf == set_ii
```

Comment:
Tests looks good! I assume these are the ones you've added? Hard to see when the automatic formatting brings so many more changes.

Reply:
I am not sure whether I understand its mathematical meaning. If the test failed, It means the math definition is wrong, then we need to ask the opinion of the author who define those functions.

Reply:
@oscargus , this part failed. I do think, I didn't get the correct mathematical definition. Do you know who is the auther of this section?

Reply:
This can be about interval arithmetic, but I guess the original code is inconsistent about giving
`FiniteSet(-oo)` and `FiniteSet({S.NegativeInfinity})`, which yields completely different result.
And neither it is working because it does not import `oo`, and it definitely needs some fix.

I think that giving `FiniteSet(-S.NegativeInfinity)` should be correct than having additional curly braces, though I'm not very sure about the difference between an open interval with infinity added with infinity, and a closed interval with infinity added with infinity.

Reply:
@sylee957 , OK, got it.